css/modals/invite/_info.scss

```diff
@@ -185,6 +185,7 @@
     font-size: 12px;
     max-height: 100%;
     overflow: auto;
+    padding: 15pt;
```

Comment:
On mobile we prefer pt, because it's relative to the dpi setting of the screen and works generally better. This setting will not affect browsers, as they have a much bigger screen than a padding would kick in.